Abhi-AnimeX/app/src/main/java/com/abhijith/animex/domain/model/AnimeItem.kt

```diff
@@ -0,0 +1,15 @@
+package com.abhijith.animex.domain.model
+
+data class AnimeItem(
+    val title: String,
+    val source: String,
+    val year: String,
+    val japaneseName: String,
+    val rating: String,
+    val rank: String,
+    val score: String,
+    val synopsis: String,
+    val genres: List<String>,
+    val imageUrl: String,
+    val youtubeId: String
```

Comment:
no default values required?

Reply:
@anandwana001  since this is a domain object and it gets assigned with values from AnimeEntity from data module using the mapper... do we still need then?